src/api/roles/admin/admin.py

```
@@ -65,10 +65,49 @@ def serialize_admin_account(account: Account) -> AdminAccountItem:
         status="active" if account.is_active else "deactivated",
         is_active=account.is_active,
+        is_suspended=account.is_suspended,
         created_at=account.created_at,
         last_active=None,
     )
 
+
+@router.post("/accounts/{account_id}/suspend", response_model=DeactivateAccountResponse)
+def suspend_account(
+    account_id: int,
+    db: Session = Depends(get_session),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)